src/nordjylland_news/image_caption_dataset.py

```diff
@@ -83,12 +87,14 @@ def build_dataset(self) -> None:
 
             self.sleep()
 
-    def get_image_data(self, article: dict) -> None:
+    def get_image_data(self, article: dict, download_images=True) -> None:
         """Gets image meta data for every image with a caption in the article.
 
         Args:
             article (dict):
                 Article data.
+            download_images (bool):
+                Whether to download images or not (used to avoid downloading images when testing).
```

Comment:
```suggestion
            download_images (bool, optional):
                Whether to download images or not (used to avoid downloading images when testing).
                Defaults to True.
```